MAUI/ImageEditor/migration.md

```diff
@@ -0,0 +1,487 @@
+
+<tr>
+<td>Rotate</td>
+<td>Rotate</td>
+<td>Method to rotate image 90 degrees in clockwise direction.</td>
+</tr>
+
+<tr>
+<td>Flip</td>
+<td>Flip</td>
+<td>Method to reverses the image along its horizontal or vertical axis.</td>
+</tr>
+
+<tr>
+<td>Crop</td>
+<td>Crop</td>
+<td>Method to select and remove a portion of the image.</td>
+</tr>
+
+<tr>
+<td>AddText</td>
+<td>AddText</td>
+<td>Method to add text to the image.</td>
+</tr>
+
+<tr>
+<td>AddShape</td>
+<td>AddShape</td>
+<td>Method to add shapes such as rectangles, circles, and lines to the image.</td>
+</tr>
+
+<tr>
+<td>Save</td>
+<td>Save</td>
+<td>Method to save the edited image.</td>
+</tr>
+
+<tr>
+<td>SaveEdits</td>
+<td>SaveEdits</td>
+<td>Method to save the current editing actions.</td>
+</tr>
+
+<tr>
+<td>CancelEdits</td>
+<td>CancelEdits</td>
+<td>Method to cancel the editing actions.</td>
+</tr>
+
+<tr>
+<td>ClearAnnotations</td>
+<td>ClearAnnotations</td>
+<td>Method to remove all the annotations.</td>
+</tr>
+
+<tr>
+<td>Reset</td>
+<td>Reset</td>
+<td>Method to reset the edited image to its initial stage.</td>
+</tr>
+
+<tr>
+<td>Undo</td>
+<td>Undo</td>
+<td>Method to reverse or undo the last action or a series of actions.</td>
+</tr>
+
+<tr>
+<td>Redo</td>
+<td>Redo</td>
+<td>Method to replay or redo an action or a series of actions that were previously undone using the "Undo" command.</td>
+</tr>
+
+<tr>
+<td>Delete</td>
+<td>DeleteAnnotation</td>
+<td>Method to remove the current selected annotations.</td>
+</tr>
+
+<tr>
+<td>GetStream</td>
+<td>GetImageStream</td>
+<td>Method to get the edited image stream</td>
+</tr>
+
+<tr>
+<td>ApplyImageEffect</td>
+<td>ImageEffect</td>
+<td>Method to apply various visual effects to the image, such as adjusted brightness, blur, contrast, exposure, saturation, sharpen, hue, and opacity.</td>
+</tr>
+
+<tr>
+<td>Bounds</td>
+<td>Bounds</td>
+<td>Gets or sets the annotation view bounds.</td>
+</tr>
+
+<tr>
+<td>EnableDrag</td>
+<td>AllowDrag</td>
+<td>Gets or sets a value indicating whether to enable drag action for annotation.</td>
+</tr>
+
+<tr>
+<td>IsResizable</td>
+<td>AllowResize</td>
+<td>Gets or sets a value indicating whether the resizing action is enabled or not for annotation.</td>
+</tr>
+
+<tr>
+<td>Opacity</td>
+<td>Opacity</td>
+<td>Gets or sets the opacity value. It ranges from 0 to 1.</td>
+</tr>
+
+<tr>
+<td>Color</td>
+<td>Color</td>
+<td>Gets or sets the shape annotation color.</td>
+</tr>
+
+<tr>
+<td>StrokeWidth</td>
+<td>StrokeThickness</td>
+<td>Gets or sets the shape annotation stroke thickness.</td>
+</tr>
+
+<tr>
+<td>FillColor</td>
+<td>IsFilled</td>
+<td>Gets or sets a value indicating whether the color should be applied to the shape fill or stroke.</td>
+</tr>
+
+<tr>
+<td>Angle</td>
+<td>RotationAngle</td>
+<td>Gets or sets the initial rotation angle for text.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>TextStyle</td>
+<td>Gets or sets the text style for text.</td>
+</tr>
+
+<tr>
+<td>Nil</td>

```

Comment:
Mention about RotatbleElements in xf

Reply:
RotatableElements property placed in the place of Nil